### PR TITLE
[Tooling] fix and add makefile targets

### PR DESCRIPTION
**Pull request by bryanchriswhite**
## Summary

- Adds the following make targets:
  - `query_tx`
  - `query_tx_json`
  - `query_tx_log`
  - `params_get_tokenomics`
  - `params_get_proof`
  - `params_get_shared`
  - `params_get_service`
- Fixes the following make targets:
  - `supplier1_unstake`
  - `supplier2_unstake`
  - `supplier3_unstake`

## Issue

- Observations while working on #799

## Type of change

Select one or more from the following:

- [ ] New feature, functionality or library
- [ ] Consensus breaking; add the `consensus-breaking` label if so. See #791 for details
- [ ] Bug fix
- [x] Code health or cleanup
- [ ] Documentation
- [ ] Other (specify)

## Testing

- [ ] **Documentation**: `make docusaurus_start`; only needed if you make doc changes
- [ ] **Unit Tests**: `make go_develop_and_test`
- [ ] **LocalNet E2E Tests**: `make test_e2e`
- [ ] **DevNet E2E Tests**: Add the `devnet-test-e2e` label to the PR.

## Sanity Checklist

- [x] I have tested my changes using the available tooling
- [x] I have commented my code
- [x] I have performed a self-review of my own code; both comments & source code
- [ ] I create and reference any new tickets, if applicable
- [ ] I have left TODOs throughout the codebase, if applicable

### CODE REVIEW

File: x/shared/types/params.go [hunk 1]
```diff
@@ -4,6 +4,7 @@ import (
 	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
 )
 
+// TODO_DISCUSS: Should these defaults be updated to match the current config.yml?
 const (
 	DefaultNumBlocksPerSession                = 4
 	ParamNumBlocksPerSession                  = "num_blocks_per_session"
```
Comment (bryanchriswhite) [inline x/shared/types/params.go:10]: @reviewer: :question:

Comment (Olshansk) [reply x/shared/types/params.go:10]: Yes

File: Makefile [hunk 2]
```diff
@@ -394,6 +394,18 @@ send_relay_sovereign_app_REST: # Send a REST relay through the AppGateServer as
 cosmovisor_start_node: # Starts the node using cosmovisor that waits for an upgrade plan
 	bash tools/scripts/upgrades/cosmovisor-start-node.sh
 
+.PHONY: query_tx
```
Comment (Olshansk) [inline Makefile:397]: Please add a `### Query Helpers ###` section

**Review by Olshansk — APPROVED** [review]

**Comment by bryanchriswhite** [comment]
Merging despite (unrelated) CI failures.

CI passed as of 747cb12 and the only subsequent commit (0a58920) adds/removes comments.
image

**Pull request merged**